draft-huque-dnsop-ns-revalidation.xml

```diff
@@ -215,7 +215,8 @@
 
       <t>
       <list style="symbols">
-
+	<!-- Ralph: I'm not sure I like the terms forwarding the triggering
+	     query, this sound very much pre-qname minimisation -->
```

Comment:
Yeah, I agree. That was Vixie's original terminology. In the qname minimization world, perhaps "resolving the triggering query" is better.